### DISCUSSION TITLE
Star lists with heirarchy

### DISCUSSION THREAD

**Discussion by joehays** [community/community#73280] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I would like to add some hierarchy to my starred repo lists. Being able to organize the lists into a tree hierarchy would be very convenient/helpful.

**Comment by oasissoman** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I would like to add some hierarchy to my starred repo lists. Being able to organize the lists into a tree hierarchy would be very convenient/helpful.

**Reply by oasissoman** [reply]
# - [ ] @oasissoman @[]()_****_

**Reply by Marcysiegel** [reply]
Hi 👋

**Comment by Marcysiegel** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I would like to add some hierarchy to my starred repo lists. Being able to organize the lists into a tree hierarchy would be very convenient/helpful.

Hi 👋

**Comment by Imadnajam** [comment]
HI  #1

**Reply by Marcysiegel** [reply]
Hello my name is marcy and you?

**Comment by Marcysiegel** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I would like to add some hierarchy to my starred repo lists. Being able to organize the lists into a tree hierarchy would be very convenient/helpful.

Hello my name is marcy and you?

**Reply by Marcysiegel** [reply]
Hello my name is marcy and you?

**Comment by 0774916089** [comment]
Safiullah Roohani
